src/pages/CreatePost.vue

```diff
@@ -113,7 +130,13 @@ export default defineComponent({
         message: this.message
       }
       console.log('posting message', entry)
-      await this.postMessage({ wallet: this.$wallet, entry, satoshis: this.offering * 1_000_000, topic: this.topic, parentDigest: this.parentDigest })
+      await this.postMessage({
```

Comment:
Should fix these lint re-formats outside of this commit. Didn't notice when I pushed.